src/main/java/com/codeborne/selenide/junit5/BrowserStrategyExtension.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 Aliaksandr Rasolka
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Selenide is licensed by MIT. Why we have here Apache 2?

Reply:
Oops. Sorry about that. Copied from another project.